server/README.md

```diff
@@ -0,0 +1,6 @@
+## Module description
+
+<p>App working with Java 8</p>
```

Comment:
It's not HTML ;-)

Reply:
I thing this README needs more love. Describe how to run the project, dependencies, it's not obvious for someone that doesn't have experience in Spring development